client/lib/user/user.js

```diff
@@ -69,6 +70,9 @@ User.prototype.initialize = function() {
 		if ( this.data ) {
 			this.clearStoreIfChanged( this.data.ID );
 			store.set( 'wpcom_user', this.data );
```

Comment:
its kind of strange that `wpcom_user` will also by default have all of the abtests instead of just the abtests key.
Do you see abtests as being a sensical part of the user object or do you think it just makes sense to piggyback on its api call + then remove it from the object before setting to localstorage?

Reply:
I'm not sure if I follow your reasoning - this block is ingesting data from user bootstrapping and making it available in local storage. It will only include the user's assigned A/B test variations instead of `all of the abtests`. (For example, my user object currently contains only one key-value pair for our running A/B tests.)

I believe having the user's assigned A/B variations in the user object as appropriate.

Reply:
so i guess my point is this.  Why store it in two places when one will do?

```jsx
const getABTests = () => store.get( 'wpcom_user ).abtests;
```